frontend/src/components/CombatResults.vue

```diff
@@ -184,9 +199,116 @@ export default Vue.extend({
 .crypto-warrior-image {
   max-width: 13em;
 }
+
+
+.bb{
+  border-bottom: 1px solid rgba(255, 255, 255, 0.282);
+  width: 70%;
+  margin: auto;
+}
+
+.bt{
+  border-top: 1px solid rgba(255, 255, 255, 0.282);
+
+}
+
+.win-details > h5{
+  font-family: Roboto;
+  color: rgba(255, 255, 255, 0.405);
+  font-size: 15px;
+  margin-bottom: 10px;
+  margin-top: 10px;
+}
+
+.earned > h4{
+  font-family: Roboto;
+  color: #fff;
+  font-size:22px;
+}
+
+.earned > h5{
+  font-family: Roboto;
+  color: rgb(186, 186, 4);
+  margin-bottom: 30px;
+}
+
+.gas-spent{
+    font-family: Roboto;
+    font-size: 15px;
+    color: rgba(255, 255, 255, 0.357);
+}
+
+.win-details > img{
+  width: 9px;
+  margin-bottom: 10px;
+  margin-top: 10px;
+}
+
+.you-earned > h4{
+  font-size: 25px;
+  font-family: 'Roboto', 'serif'  !important;
+}
+
 .ad-container {
   display: flex;
   flex-direction: column;
   gap: 0.5rem;
 }
+
+.top-bg-img > img, .bot-bg-img > img {
+    width: 69% !important;
+    margin-bottom: -70px;
+
+}
+
+.modal-body > div{
+  width: 600px !important;
+}
+
+
+.tob-bg-img > img{
+  margin-top: -40px !important;
+}
+
+.modal-content{
+  background-color: rgba(255, 255, 255, 0) !important;
+}
+
+.modal-body{
+  background: linear-gradient(to right, rgb(20 20 20 / 10%), rgba(20,20,20,80%),rgb(20 20 20 / 100%),
+  rgb(20 20 20 / 80%),rgb(20 20 20 / 40%),rgb(20 20 20 / 10%)) !important;
+  text-align: center !important;
+  width: 630px !important;
+}
+
+.modal-dialog{
+  margin-top: 5em;
+    display: flex !important;
+    align-items: center !important;
+    justify-content: center !important;
+}
+
+@media all and (max-width: 600px) {
+  .modal-dialog{
+    margin-top: 10em;
+  }
+
+  .outcome{
+    font-size: 1.5em;
+  }
+
+  .modal-content{
+    width: auto !important;
+  }
+
+  .float-center .container{
+    padding: 0px !important;;
+  }
+
+  .float-center .container .bb{
+    padding: 0px !important;
+    width: 100% !important;
+  }
+}
```

Comment:
Same as above comment just needs a tidy up and thinking about different screen sizes. The overuse of !important shows that there is conflicting css you are overriding